assignments/callbacks.js

```diff
@@ -21,32 +21,63 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
+  
+  return cb(arr.length)
```

Comment:
You don't have your Call back defined. You need to to to write that function. something like

```javascript
function logToConsole(item) {
	console.log(item);
}

getLength(items, logToConsole);
```